ui-v2/config/environment.js

```diff
@@ -136,7 +129,7 @@ module.exports = function(environment, $ = process.env) {
         // won't really exist in the actual ember ENV when it's being served
         // through Consul. See settingsInjectedIndexFS.Open in Go code for the
         // details.
-        CONSUL_UI_SETTINGS_PLACEHOLDER: "__CONSUL_UI_SETTINGS_GO_HERE__",
+        CONSUL_UI_SETTINGS_PLACEHOLDER: '__CONSUL_UI_SETTINGS_GO_HERE__',
```

Comment:
sadly, I think this change will break our injecting code... It looks for `%22` which is double quotes. I had some ideas for making this less brittle, but I don't know ember well enough to know how we can implement them.

Reply:
Thanks for calling this out! I'll change it back, I think it was picked up by VScode on-save formatting and I didn't mean to change this myself.